IDEAS/Fluid/Movers/Data/Pumps/Wilo/Stratos25slash1to4.mo

```diff
@@ -17,34 +17,12 @@ record Stratos25slash1to4 "Pump data for a Wilo Stratos 25/1-4 pump"
 defaultComponentPrefixes="parameter",
 defaultComponentName="per",
 Documentation(info="<html>
-  <p>Data from:
-  <a href=\"http://productfinder.wilo.com/en/COM/product/0000000e000379df0002003a/fc_product_datasheet\">
-  http://productfinder.wilo.com/en/COM/product/0000000e000379df0002003a/fc_product_datasheet
-  </a>
-  </p>
-  <p>See
-  <a href=\"modelica://IDEAS.Fluid.Movers.Data.Pumps.Wilo.Stratos25slash1to6\">
-  IDEAS.Fluid.Movers.Data.Pumps.Wilo.Stratos25slash1to6
-  </a>
-  for more information about how the data is derived.
-  </p>
-  </html>",
+<p>Data from: <a href=\"http://productfinder.wilo.com/com/en/c0000001b0000accb00010023/_0000000e000379df0002003a/product.html\">http://productfinder.wilo.com/com/en/c0000001b0000accb00010023/_0000000e000379df0002003a/product.html</a></p>
```

Comment:
@icupeiro you changed the documentation such that the code is on one line instead of multiple.  As you can see, this complicates peer review since not the entire line is visible. Therefore in IBPSA we have the convention to split the text across multiple lines as is shown in the original version of the code. Can you split the documentation across multiple lines again for this record and the new ones? Thanks!

Reply:
working on it
any further comments for the Veroline32/160 model? should I include a screenshot of the INFRAX technical sheet, or that should be treated as confidential information confidential from the building?

Reply:
Well, it says data not available, but then where did you get the data?